### PR TITLE
rm: ota: config: add information on device group and unsafe handlers

### PR DESCRIPTION
**Pull request by vanmaegima**
Previous version didn't include information on device group configurations, so include this.

Previous example is now considered unsafe by fioconfig, so provide information about this and steps to get a new working example.

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 95e815767f46140ec5dc306c5fe15ecc1a5e68d4 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2073/docs/artifacts/html/index.html

**Review by doanac — COMMENTED** [review]
nice. thanks

**Review by Tim-Anderson — APPROVED** [review]
LGTM

File: source/reference-manual/ota/configuring.rst [hunk 1]
```diff
@@ -3,27 +3,28 @@
...
 
-Device configuration can be managed with the `fioctl command line tool`_.
-There are two types of configuration supported:
+Device configuration can be managed with :ref:`ug-fioctl`.
+There are three types of configuration supported:
 
-  #. Fleet-wide - Configuration set here gets sent to all devices in a Factory.
-  #. Device specific - This overrides fleet-wide configuration in the
-     event they collide.
+  #. Fleet-wide - Configuration gets sent to all devices in a Factory.
```
Comment (kprosise) [inline source/reference-manual/ota/configuring.rst:9]:
```suggestion
  #. Fleet-wide — Configuration gets sent to all devices in a Factory.
```

File: source/reference-manual/ota/configuring.rst [hunk 2]
```diff
@@ -3,27 +3,28 @@
...
+  #. Device group specific - Configuration gets sent to all devices in a device group.
```
Comment (kprosise) [inline source/reference-manual/ota/configuring.rst:10]:
```suggestion
  #. Device group specific — Configuration gets sent to all devices in a device group.
```

**Review by kprosise — APPROVED** [review]
minor style suggestions, otherwise it LGTM and is otherwise ready!

**Comment by doanac** [comment]
Docs for 0c24ca63a6b7a8e66e2deac7c42bae81b53b6993 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2075/docs/artifacts/html/index.html

**Pull request merged**